Week02/lesson03/classwork/tags_boxes/css/style.css

```diff
@@ -0,0 +1,33 @@
+h1{
```

Comment:
SUPER nitpicky of me, but add a space between the selector `h1` in this case and the `{` as it's industry convention and makes the file easier to read.